modules/virt-reclaiming-persistent-volumes-for-binding-to-a-new-persistent-volume.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// virt/virtual_machines/virtual_disks/virt-reclaiming-persistent-volumes.adoc
+
+[id="virt-reclaiming-persistent-volumes-for-binding-to-a-new-persistent-volume_{context}"]
+
+= Reclaiming persistent volumes for binding to a new persistent volume
+
+Use this procedure to:
+
+* Release the persistent volume from the persistent volume claim.
+* Erase all data in the share.
+* Create a new persistent volume with a different name,
+maintaining the persistent volume's original connection settings.
+
+.Prerequisite
+* Note the connection details of the persistent volume you want to reclaim.
+
+.Procedure
+. Ensure that the Storage Class *reclaimPolicy* is set to *delete*.
+. Verify that no other resources are using the persistent volume claim, then
+delete the persistent volume claim to release the persistent volume.
++
+----
+$ oc delete pvc/<volume claim>
+----
++
+. Delete the persistent volume.
++
+----
+$ oc delete pv/<volume>
+----
++
+. Delete the data in the share.
++
+----
+$ rm -rf <path_to_share>
+----
++
+. Create a new persistent volume with a different volume name. When creating the
```

Comment:
~a new~
s/with/that uses

How do you know that you have the right connection details?

I'd provide the commands to create the PV, include a modules that describes creating a PV, or provide an assembly-level "next steps" section with a link to an assembly about creating a PV.

Reply:
I brought up the idea of adding the command to create the PV but these can be lengthy and the other CNV writers didn't think it was needed. I can provide a link in the assembly under Additional Resources to the Create VM Page, which offers a number of ways to configure new disks. If something more precise is needed, I will open a doc bug for next release.

Reply:
@bgaydosrh, you can't add an xref to a module. It doesn't work right in the current FCC implementation, though it is something that the tooling team is trying to fix for Pantheon 2. If your modules still contain xrefs, you need to remove them. Please file an RFE to fix them if they still exist. (@vikram-redhat, FYI)

Reply:
@ousleyp, @aburdenthehand, what do you think about adding an explicit step for creating a PV here?

Reply:
@kalexand-rh - Yeah, noted - was trying to add the Xref to the assembly.

Reply:
> @ousleyp, @aburdenthehand, what do you think about adding an explicit step for creating a PV here?

+1, the user would probably have to go searching for instructions at this point if you leave it as-is.

Reply:
Would pointing the user to the Create VM assembly help --- that is where they can create disks, edit configs, right? If someone can point me to the correct content, I will be glad to either copy or link to it in the assembly.

Reply:
Added steps per dev.